source/atlas-cli-changelog.txt

```diff
@@ -32,6 +32,10 @@ Released 9 June 2023
 
 Released 7 June 2023
 
+- Introduces a breaking change to the :ref:`atlas-alerts-settings-list`
+  command that modifies its ``--output json`` response structure.
+  Use the nested ``.Results`` object to access the previous response
+  structure.
```

Comment:
using changelog standard for breakingchanges

```suggestion

BREAKING:

Commands <list> return different JSON format. 
Instead of x return y. 
To migrate your script we recommend:

jq ... => jq ... 


- Introduces a breaking change to the :ref:`atlas-alerts-settings-list`
  command that modifies its ``--output json`` response structure.
  Use the nested ``.Results`` object to access the previous response
  structure.
```

A

Reply:
@wtrocki thanks for the suggestion. Currently we don't use the breaking changes standards in the product release notes -- if we switched in this PR it wouldn't be aligned with the release notes for the rest of our products. I'm open to this, but I think the docs team should consider the ramifications of this before adopting this standard in release notes. I'll add this as an agenda item for our next sync with APIx.

Reply:
no worries. I was suggesting some edit to @matt-condon